### PR TITLE
Editorial review edits

### PR DESCRIPTION
**Pull request by EMcWhinn**
https://issues.redhat.com/browse/AAP-15796

### CODE REVIEW

File: downstream/modules/hub/proc-add-user-to-group.adoc [hunk 1]
```diff
@@ -15,8 +14,8 @@ Complete the following steps to add users to an existing group.
 
 . Log in to your local {HubName}.
 . From the navigation panel, select menu:User Access[Groups].
-. Click on a Group name.
-. Navigate to the menu:Users[] tab, then click btn:[Add].
+. Click on a *Group* name.
+. Navigate to the *menu:*Users* tab and click btn:[Add].
```
Comment (michellemacrh) [inline downstream/modules/hub/proc-add-user-to-group.adoc:18]: Is there an extra asterisk (*) here i.e. 3 when there should be 2?

File: downstream/modules/hub/proc-add-user-to-group.adoc [hunk 2]
```diff
@@ -15,8 +14,8 @@ Complete the following steps to add users to an existing group.
...
 . Select users to add from the list and click btn:[Add].
 
-When you add the selected users to the group, they have permissions to use {HubName} assigned to the group.
+When you add the selected users to the group, they have the permissions to use {HubName} that are assigned to the group.
```
Comment (michellemacrh) [inline downstream/modules/hub/proc-add-user-to-group.adoc:21]: It looks like {HubName} evaluates to "automation hub". If that's the case the sentence reads "they have the permissions to use automation hub that are assigned to the group." Should automation hub be automation hubs?

File: downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc [hunk 3]
```diff
@@ -62,11 +62,12 @@ name of your key file.
 +
 . Save and close the file.
 
-. Pull the file using podman, or your client of choice:
+.Verification
+* Pull the file using podman, or your client of choice:
```
Comment (michellemacrh) [inline downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc:66]: I think podman should be capitalized if it refers to the application? Tho it's not in the Supplemental Style Guide so I can't be sure 🤔

**Review by michellemacrh — COMMENTED** [review]
Only one or two small notes but otherwise LGTM :)

File: downstream/modules/hub/proc-add-user-to-group.adoc (hunk 2 above)
Comment (EMcWhinn) [reply downstream/modules/hub/proc-add-user-to-group.adoc:21]: Yes the phrasing is awkward here, but it refers to users having the "permissions" assigned to that group in hub. I think it should be rephrased for clarity.

File: downstream/modules/hub/proc-add-user-to-group.adoc (hunk 1 above)
Comment (EMcWhinn) [reply downstream/modules/hub/proc-add-user-to-group.adoc:18]: Fixed.

File: downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc (hunk 3 above)
Comment (EMcWhinn) [reply downstream/modules/hub/proc-configuring-the-client-to-verify-signatures.adoc:66]: Fixed.

File: downstream/modules/hub/proc-add-user-to-group.adoc (hunk 2 above)
Comment (michellemacrh) [reply downstream/modules/hub/proc-add-user-to-group.adoc:21]: Ah thanks for clarifying 👍 agreed!

**Review by michellemacrh — APPROVED** [review]
LGTM!

**Pull request merged**